<launch>
  <arg name="base_link" default="base_link"/>
  <arg name="lever" default="0.05"/>

  <!-- load pids_values and others-->
  <rosparam file="$(find espdrone_controller)/params/controller.yaml" />

  <node name="twist_controller" pkg="espdrone_controller" type="twist_controller" output="screen">
  </node>

  <node name="motor_controller" pkg="espdrone_controller" type="motor_controller" output="screen">
    <param name="lever" value="$(arg lever)"/>
    <param name="base_link_frame" value="$(arg base_link)"/>
  </node>

  <node name="drone_teleop" pkg="espdrone_controller" type="teleop_twist_keyboard.py" output="screen"/>

</launch>